client/components/editor.vue

```diff
@@ -24,6 +24,27 @@
         </v-card-actions>
       </v-card>
     </v-dialog>
+
+    <v-dialog hide-overlay v-model="commentDialog" max-width="500px">
```

Comment:
Didn't we say that you will not use a dialog but that you will make it like Google Doc that an entry form is shown on the right so that it does not overlay the content?

Reply:
Right, I see you opened #44 to make that change.